modules/data-streams/src/main/java/org/elasticsearch/datastreams/lifecycle/DataStreamLifecycleService.java

```diff
@@ -797,7 +831,8 @@ private Set<Index> maybeExecuteRollover(ClusterState state, DataStream dataStrea
                         currentRunWriteIndex.getName(),
                         errorStore,
                         Strings.format(
-                            "Data stream lifecycle encountered an error trying to rollover data steam [%s]",
+                            "Data stream lifecycle encountered an error trying to rollover[%s] data steam [%s]",
+                            rolloverFailureStore ? " the failure store of the" : "",
```

Comment:
I prefer to keep it rollover. I know it's not the correct use of the phrasal verb, but we use it a lot like this and it makes the logs more discoverable because you can search for `rollover` and find all the relevant logging.

Reply:
Hmm I see plenty of usages of `roll over` too, and searching for `roll over` should be just as easy/doable as `rollover`, right?

Reply:
Well I find it much easier to drop the word `rollover` in the kibana query than writing message should be rollover or "roll over". However, as you pointed out in the code we find both log lines that use the action as verb `rollover` and ones that use the grammatically correct `roll over`. In this class we use everywhere the former. I will see how big of a change is to switch them all, if it's easy I will do it. If not, then we should have a discussion align and open a different PR in which we align the code. So, stay tuned :)